tests/Integration/AbstractTechproductsTest.php

```diff
@@ -1499,7 +1499,8 @@ public function testAnonymouslyNestedDocuments()
         }
 
         // in Solr 7, atomic updates of child documents aren't possible
-        if (8 <= self::$solrVersion) {
+        // in SolrCloud mode, this fails more often with "Async exception during distributed update" than it succeeds
```

Comment:
This distributed stuff isn't easy. Maybe we need to do some more adjustments in the solrconfig.
One thing I didn't check yet for example is if the 'stats_cache' is set 'org.apache.solr.search.stats.LRUStatsCache'.

Reply:
It isn't set, but it doesn't make a difference if I set it.

If you think it is useful in general, I can open a different PR for that patch since I wrote it anyway.

Reply:
Yes, we should open a PR for "distributed search" config stuff. I did some adjustments for search_api_solr we should review and adopt.

So I suggest that we add a todo comment to the if statement referencing the new PR. And then we should commit this PR.

Reply:
I've opened issue #908 for it. I'm not familiar with search_api_solr so I'll leave the PR to you. 😉